website/api/controllers/webhooks/receive-from-stripe.js

```diff
@@ -69,12 +69,14 @@ module.exports = {
     let subscriptionForThisEvent = await Subscription.findOne({stripeSubscriptionId: subscriptionIdToFind}).populate('user');
 
     let STRIPE_EVENTS_SENT_BEFORE_A_SUBSCRIPTION_RECORD_EXISTS = [
-      'invoice.created',
-      'invoice.finalized',
-      'invoice.paid',
-      'invoice.payment_succeeded',
-      'invoice.payment_failed',
-      'invoice.payment_action_required',
+      'invoice.created',// Sent when a user creates their subscription
+      'invoice.finalized',// Sent when a user creates their subscription
```

Comment:
Both are sent at the same time when a user creates their subscription?

How does a user create their subscription?  What human action is happening, in the contest of Fleet?

I love the idea to add comments, but this reads like we copy and pasted it from somewhere.